content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -9,176 +9,268 @@ With a down payment, the buyer pays a portion of the total amount owed while agr
 remaining amount at a later date. In turn, the seller provides goods or services to the buyer after
 accepting the down payment, trusting that the remaining amount will be paid later on.
 
-Initial down payment request
-============================
+Create invoices
+===============
 
```

Comment:
Do you think it would add value to this doc if we included brief instructions on creating and confirming a sales order here, before the invoice section? Or is that implied/not necessary in this particular doc?

Reply:
I understand this suggestion, however, I feel like if a reader is interested in this doc, they will already know how to accomplish those sales order goals. So, I think I'll leave this addition out for now -- but if readers start to reach out, wondering how they can do those things, I'll be sure to add it in, per your request. Thanks for bringing this to my attention. I appreciate it! 👍